tests/variable_test.py

```diff
@@ -0,0 +1,353 @@
+import defcon
```

Comment:
you need not import defcon directly here. You can use the `ufo_module` global fixture defined in tests/conftest.py and already available to all pytest tests under tests/ folder. That way tests will be run twices, once with defcon, the other time with ufoLib2 (so you test both).
Actually you only import defcon module to initialise a Font object, so you want to use the `FontClass` fixture also defined inside tests/conftest.py module.
Note that you don't need to explicitly import the fixture in conftest.py, that's a special module that pytest looks for and automatically makes all fixture defined in it available for tests defined in the same folder tree.
grep for `FontClass` in the tests/ folder to see examples of how it's used.

Reply:
I fixed this in b0e26d67ded01eccb22ea87eb53e459475b964b6